ucey-star/ID8
# Project_idea hookup

## Description

This PR hookups up the project idea page with the back-end. I have removed the component from the app directory and created the form as a component in the `components` folder fixing issue https://github.com/ucey-star/ID8/issues/27. This issue is now closed. 

## Type of Change


## Testing Instructions
1. Ensure you have the environment variables for supabase in you env.local 
2. Checkout into this branch 
3. from the url go to the `http://localhost:3000/project_idea` and input data into the form. If its a success you should see the snackbar showcasing success. 


## Screenshots (if applicable)

If your changes include visual components, add screenshots showing the affected pages or elements.
